easybuild/easyconfigs/s/sbt/sbt-1.6.2-Java-8.eb

```
@@ -0,0 +1,25 @@
+easyblock = 'Tarball'
+
+name = 'sbt'
+version = '1.6.2'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'http://www.scala-sbt.org/'
+description = "sbt is a build tool for Scala, Java, and more."
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/sbt/sbt/releases/download/v%(version)s']
+sources = [SOURCE_TGZ]
+checksums = ['854154de27a7d8c13b5a0f9a297cd1f254cc13b44588dae507e5d4fb2741bd22']
```

this is the checksum for 1.3.13 instead of 1.6.2
I get a checksum of `637637b6c4e6fa04ab62cd364061e32b12480b09001cd23303df62b36fadd440` instead 
```suggestion
checksums = ['637637b6c4e6fa04ab62cd364061e32b12480b09001cd23303df62b36fadd440']
```